src/components/analyses/ShareWithSupportDialog.js

```diff
@@ -50,36 +51,32 @@ function AnalysisInfo(props) {
     if (analysis) {
         return (
             <Grid container spacing={3} style={{ marginTop: 2, fontSize: 12 }}>
```

Comment:
Sorry missed this one. `marginTop` should use `theme.spacing()`. `fontSize` should not be set.

Reply:
I think it looks good without the `margin` as well, so I'll just remove the custom styling here too:
![Analysis Summary - no margin top](https://user-images.githubusercontent.com/996408/94748269-c3ea6300-0335-11eb-87f8-b56ffedda4ec.png)

Reply:
Just a quick note - `GridLabelValue` doesn't assume that the value, i.e. children passed in, will always be text like the `label` does, so you might need to wrap the values in a `Typography`.  Looks like the font is different between the two right now.

Reply:
Good catch. I never seem to notice the font difference 😖 
(probably because I generally prefer serif fonts)